common/activation.py

```diff
@@ -0,0 +1,12 @@
+''' Number of activation functions '''
+import autograd.numpy as np
+
+identity = lambda x: x
+
+tanh = lambda x: np.tanh(x)
+
+sigmoid = lambda x: 0.5*(np.tanh(x) + 1.0)
+
+rbf = lambda x: np.exp(-x**2)
```

Comment:
RBF stands for a radial basis function. How neural networks generally works is that they try to approximate very complicated functions using a number of sequentially stacked non-linear functions. For example, a sigmoid function is a popular one (https://qph.ec.quoracdn.net/main-qimg-05edc1873d0103e36064862a45566dba?convert_to_webp=true): it maps a continuous value to a value between 0 and 1. RBF is just another possible non-linear function. Often it's related to a Gaussian distribution but you should think of it as a way to transform data into a "more interesting" space.